source/frontend/src/features/mapSideBar/acquisition/tabs/agreement/models/AcquisitionAgreementFormModel.ts

```diff
@@ -0,0 +1,89 @@
+import { ApiGen_Concepts_Agreement } from '@/models/api/generated/ApiGen_Concepts_Agreement';
+import { stringToNull, stringToNumber, toTypeCodeNullable } from '@/utils/formUtils';
+
+export class AcquisitionAgreementFormModel {
+  public agreementTypeCode: string | null = '';
+  public agreementTypeDescription: string | null = '';
+  public agreementDate: string | null = '';
+  public completionDate: string | null = '';
+  public terminationDate: string | null = '';
+  public commencementDate: string | null = '';
+  public possessionDate: string | null = '';
+  public depositAmount: string | null = '';
+  public noLaterThanDays: string | null = '';
+  public purchasePrice: string | null = '';
+  public legalSurveyPlanNum: string | null = '';
+  public offerDate: string | null = '';
+  public expiryDateTime: string | null = '';
+  public signedDate: string | null = '';
+  public inspectionDate: string | null = '';
+  public agreementStatusTypeCode: string | null = 'DRAFT';
+  public agreementStatusTypeDescription: string | null = '';
+  public cancellationNote: string | null = '';
+  public isDraft: boolean | null = null;
```

Comment:
all these are marked as nullable, but never actually set to null. Should they just be set to string instead?

Reply:
updated